insights/parsers/qemu_xml.py

```diff
@@ -1,13 +1,51 @@
 """
-QemuXML file ``/etc/libvirt/qemu/*.xml``
-----------------------------------------
+QemuXML file ``/etc/libvirt/qemu/*.xml`` and ``/var/run/libvirt/qemu/*.xml``
+----------------------------------------------------------------------------
+
+Parsers provided by this module are:
+
+QemuXML - file ``/etc/libvirt/qemu/*.xml``
+------------------------------------------
+
+VarQemuXML - file ``/var/run/libvirt/qemu/*.xml``
+-------------------------------------------------
 """
 from .. import XMLParser, parser
 from insights.specs import Specs
 
 
+class BaseQemuXML(XMLParser):
+    """Base class for parsing Qemu XML files. It uses ``XMLParser`` mixin
+    class.
+
+    """
+    def parse_content(self, content):
+        super(BaseQemuXML, self).parse_content(content)
+
+        # Setting vm_name attribute
+        self.vm_name = None
+        ret = self.get_elements('./domain/name') if self.dom else None
+        if ret:
+            self.vm_name = ret[0].text
+        else:
+            self.vm_name = self.data.get('name')
+
+    def parse_dom(self):
```

Comment:
Please add a docstring to this method so that it doesn't use the default docstring from the base class.

Reply:
Done